python/docs/source/conf.py

```diff
@@ -93,7 +93,7 @@ def setup(app):
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'default'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
@peterychang @gramhagen thoughts on changing to read the docs theme? Is this something we want to do?

Reply:
no strong preference, one con is it requires additional dependency to install?